nemo/collections/asr/models/ctc_models.py

```diff
@@ -514,6 +522,58 @@ def training_step(self, batch, batch_nb):
 
         return {'loss': loss_value, 'log': tensorboard_logs}
 
+    def stream_step(
+        self,
```

Comment:
1) Add docstring and explain what this is.
2) Move it to ASRModuleMixin, check that module inherits ConformerEncoder (and if necessary ctc/rnnt decoder by seeing if module.joint exists or not)
3) If it exists, then great, perform this step.
4) Document the steps here (one line is fine).
5) Add types to your function inputs if possible.

Reply:
done.